Package.swift

```diff
@@ -125,6 +144,19 @@ let package = Package(
                 .copy("PrivacyInfo.xcprivacy")
             ]
         ),
+        .target(
+            name: "SmithySwiftNIO",
+            dependencies: [
+                "Smithy",
+                "SmithyHTTPAPI",
+                "SmithyHTTPClient",
```

Comment:
`SmithyHTTPClient` depends on CRT, which means that `SmithySwiftNIO` depends on CRT.

We need to find a way to break that dependency.